source/slides/11_chef.rst

```diff
@@ -0,0 +1,596 @@
+.. _11_chef:
+
+===========
+Chef Basics
+===========
+
+Terminology
+===========
+
+*Client*
+    A client is identified by a key pair, used for authorization.
```

Comment:
You might see how this looks using [definition lists](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists). Sometimes the formatting gets weird on the slides, but I generally like how it looks.